src/librustc_interface/util.rs

```diff
@@ -108,6 +108,7 @@ pub fn create_session(
     let codegen_backend = get_codegen_backend(&sess);
 
     rustc_lint::register_builtins(&mut sess.lint_store.borrow_mut(), Some(&sess));
+    rustc_lint::register_internals(&mut sess.lint_store.borrow_mut(), Some(&sess));
```

Comment:
cc https://github.com/rust-lang/rust/pull/58701#discussion_r261378083

which flag should I use here? The `-Zunstable_options`?

Reply:
jup